regex-assembly/933100.ra

```diff
@@ -11,6 +11,7 @@
     x[^m]
     xm[^l]
     xml[^\s]
+    xml\s*[^a-z]
```

Comment:
Add comment here about what we're actually looking for, which is the label marker `:`: `xml :echo`.

Reply:
Resolved, thank you.